actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkMigrationActor.java

```diff
@@ -413,6 +413,16 @@ private String getCreatedBy(Request request) {
               break;
             case "error type":
               mappedColumns.add(JsonKey.ERROR_TYPE);
+              break;
```

Comment:
Add the "district" as Json key. Skip columns to be made configurable. Externalise the columns with including Ref PR: https://github.com/project-sunbird/sunbird-lms-service/pull/526

Reply:
Sure, we will take care this in release-3.4.0 for this.